### PR TITLE
Add end chat button in chat page

### LINKED ISSUE #31: Add a end chat button on chat page.
Please add a "End chat" button on the chat page. You can add it at the bottom of page or in the top navbar.

### PR DESCRIPTION
**Pull request by jainkeval**
**Fixes #31**

**You may view the changes here:**
 https://murmuring-badlands-63071.herokuapp.com/chat

### CODE REVIEW

**Comment by sumedh123** [comment]
@jainkeval Deployment link please.

**Comment by jainkeval** [comment]
[https://murmuring-badlands-63071.herokuapp.com](url)

File: views/chat.hbs [hunk 1]
```diff
@@ -43,9 +43,13 @@
   </div>
   <div class="ui raised segment sixteen column row full bottom">
     <div class="ui action icon input full">
+      <button class="ui red right labeled icon button" id="endbtn">
+        <i class="ban icon"></i>
+        End
+      </button>
       <input type="text" placeholder="What are you sending..." id="message">
```
Comment (sumedh123) [inline views/chat.hbs]: can you chagne the placeholder to "Enter the message here."

File: views/chat.hbs [hunk 2]
```diff
@@ -43,9 +43,13 @@
...
       <i class="smiley icon"></i>
```
Comment (sumedh123) [inline views/chat.hbs]: Remove this line. Seems of no use here.

**Pull request merged**